src/main/scala/exercises/03_Exponent.scala

```diff
@@ -14,6 +14,14 @@ object Exponent {
    * NOTE: Scala has a built-in exponent function, but you're not allowed to
    * use it to implement `expt`
    */
-  def expt(n: Int, e: Int): Int = -1
+  def expt(n: Double, e: Int): Double =
+      if (e == 0)
+        1
+      else if (n == 0)
+        0 
+      else if (e < 0)
+        1 / expt(n, e)      
+      else
+        n * expt(n, e-1)
 
```

Comment:
I like the spacing in this function and in fib.  It makes the code exponentially more readable than what I did.